datafiles/main/chapters/template.JSON

```diff
@@ -0,0 +1,302 @@
+{
+    "chapter": {
+        "id": 1,
+        "name": "Chapter Name",
+        "flavor": "Chapter Lore",
+        "origin": 3, // 1 - Founding, 2 - Successor, 3 - Other/non-canon/fanmade, 4 - Custom
+        "points": 150,
+        "founding": 0, // The id of the founding chapter, 0 for unknown or none, 10 for random
```

Comment:
I hope we will be able to choose traitor and 2 lost legions (2nd and 11th) as options, some day.
Chimeric as option too.